app/views/invitation_mailer/invite.html.erb

```diff
@@ -0,0 +1 @@
+<p>Hello has invited you to join Brownfield of Dreams.You can create an account with me <%= link_to 'here', 'https://brownfield-of-dreams-1811.herokuapp.com/register'%></p>
```

Comment:
I'm confused about this invite_mailer - Brownfield stuff.

Reply:
Just removed those lines